tmail-backend/rate-limiter/rate-limiter-api/src/test/scala/com/linagora/tmail/rate/limiter/api/RateLimitingPlanUserRepositoryContract.scala

```diff
@@ -12,8 +12,8 @@ import reactor.core.scala.publisher.{SFlux, SMono}
 import scala.jdk.CollectionConverters._
 
 object RateLimitingPlanUserRepositoryContract {
-  val BOB: Username = Username.of("BOB")
-  val ALICE: Username = Username.of("ALICE")
+  val BOB: Username = Username.of("BOB@domain1.tld")
```

Comment:
I feel it is incorrect changes,
 are u sure?

Reply:
as I remember, postgres rls test fail when these username don't have domain